Dockerfile-18

```diff
@@ -120,9 +120,13 @@ RUN fakechroot fakeroot chroot $CHROOT_DIR \
 RUN fakechroot fakeroot chroot $CHROOT_DIR \
   /bin/sh -c 'curl -s -L https://github.com/jgm/pandoc/releases/download/1.19.2.1/pandoc-1.19.2.1-1-amd64.deb -o pandoc.deb && dpkg -i pandoc.deb && rm pandoc.deb'
 
+# set default CRAN mirror and set working directory
+RUN fakechroot fakeroot chroot $CHROOT_DIR \
+   /bin/sh -c "echo 'options(repos = c(CRAN = '\''https://cloud.r-project.org/'\''), download.file.method = '\''libcurl'\'', Ncpus = parallel::detectCores(logical=FALSE)); if (dir.exists('\''/app'\'')) setwd('\''/app'\'')' >> /etc/R/Rprofile.site"
+
 # install dash 
 RUN fakechroot fakeroot chroot $CHROOT_DIR \
-  /usr/bin/R -e "r <- getOption('repos'); r['CRAN'] <- 'http://cloud.r-project.org'; options(repos=r, Ncpus=2); install.packages(c('devtools', 'callr', 'data.table', 'plotly', 'ggplot2', 'scales', 'httr', 'jsonlite',  'magrittr', 'digest', 'viridisLite', 'base64enc', 'htmltools', 'htmlwidgets', 'tidyr', 'hexbin', 'RColorBrewer', 'dplyr', 'tibble', 'lazyeval', 'rlang', 'crosstalk', 'purrr', 'promises', 'R6', 'shiny', 'assertthat', 'glue', 'pkgconfig', 'Rcpp', 'tidyselect', 'BH', 'plogr', 'gtable', 'MASS', 'mgcv', 'reshape2', 'withr', 'lattice', 'yaml', 'curl', 'mime', 'openssl', 'later', 'labeling', 'munsell', 'cli', 'crayon', 'fansi', 'pillar', 'ellipsis', 'stringi', 'vctrs', 'lifecycle', 'nlme', 'Matrix', 'colorspace', 'askpass', 'utf8', 'plyr', 'stringr', 'httpuv', 'xtable', 'sourcetools', 'backports', 'zeallot', 'sys', 'fiery', 'uuid', 'future', 'reqres', 'globals', 'listenv', 'urltools', 'brotli', 'xml2', 'webutils', 'codetools', 'triebeard')); remotes::install_github('plotly/dashR', upgrade=TRUE)" 
+  /usr/bin/R -e "install.packages(c('devtools', 'callr', 'data.table', 'plotly', 'ggplot2', 'scales', 'httr', 'jsonlite',  'magrittr', 'digest', 'viridisLite', 'base64enc', 'htmltools', 'htmlwidgets', 'tidyr', 'hexbin', 'RColorBrewer', 'dplyr', 'tibble', 'lazyeval', 'rlang', 'crosstalk', 'purrr', 'promises', 'R6', 'shiny', 'assertthat', 'glue', 'pkgconfig', 'Rcpp', 'tidyselect', 'BH', 'plogr', 'gtable', 'MASS', 'mgcv', 'reshape2', 'withr', 'lattice', 'yaml', 'curl', 'mime', 'openssl', 'later', 'labeling', 'munsell', 'cli', 'crayon', 'fansi', 'pillar', 'ellipsis', 'stringi', 'vctrs', 'lifecycle', 'nlme', 'Matrix', 'colorspace', 'askpass', 'utf8', 'plyr', 'stringr', 'httpuv', 'xtable', 'sourcetools', 'backports', 'zeallot', 'sys', 'fiery', 'uuid', 'future', 'reqres', 'globals', 'listenv', 'urltools', 'brotli', 'xml2', 'webutils', 'codetools', 'triebeard')); remotes::install_github('plotly/dashR', upgrade=TRUE)" 
```

Comment:
Can you explain this change as well?

Reply:
I'll reconfirm the package list later this evening, but the mirror setting was moved to an earlier line in `Dockerfile-18`.

https://github.com/plotly/heroku-buildpack-r-build/blob/65ec54d8065acd955c22652a91b3cbde0a758e21/Dockerfile-18#L123-L125

Reply:
@josegonzalez Sorry for the delay, but I confirm that the package list is the same. We've added `lubridate`, and specified that we should deploy the release version from `master` in lieu of deploying with `dev`.